README.md

```diff
@@ -17,20 +17,30 @@ Welcome to the the Volunteering App Github repo
 4. Make sure that you have Android 10 installed and not higher.
 
 
-# Setup     
+# Setup  
 
 1. Ensure that you've gone through the following link for your particular platform: https://reactnative.dev/docs/environment-setup
 
 2. Pull the code from Git
  
-3. Open Command terminal       
- 
-4. Go to the `app` folder inside the project folder (e.g. **\Volunteer-app path\app**)      
+3. Open Command terminal
+
+4. Go to the `api` folder inside the project folder (e.g. **\Volunteer-app path\api**)
+
+5. Create .env file to store Jira API access credentials.
+    >**Note** Credentials themselves not provided, these should be requested/provided on joining the dev group.
+
+5. At the command prompt type `npm install` then `npm start` to start the Volunteer App API server.
+
+6. Go to the `app` folder inside the project folder (e.g. **\Volunteer-app path\app**)      
  
-5. type in command terminal: npm install     
-    >**Note:** Inside the `app` folder there is package-lock.json. Everytime this is modified, it is advised to do step 4      before       running the project.      
+6. At the command prompt type `npm install`     
+    >**Note:** Inside the `app` folder there is package-lock.json. Everytime this is modified, it is advised to repeat step 5 before  running the project.
```

Comment:
I think `repeat step 5` referred to `npm install` in the `app` folder, which has now changed to a different step number so this needs to be updated?  (Maybe to make it easier to keep this up to date in the future as step numbers change, could avoid giving the step number itself and just make clear what needs to be done?)

Reply:
Updated.